database/index.js

```diff
@@ -39,62 +39,23 @@ const createWatchInfo = (watchId, watchName, options, callback) => {
 
 const getWatchInfo = (watchId, callback) => {
 
-  // SELECT name, pathfilename
-  // FROM table1
-  // NATURAL JOIN table2
-  // NATURAL JOIN table3
-  // WHERE name = 'John';
+/*
+explain analyze select t1.*, t2.wid, t3.strap_id, t3.strap_name, t3.strap_image, t3.strap_price
```

Comment:
Wasn't explain analyze just to time test queries locally? You may want to look at that!